### DISCUSSION TITLE
What is the meaning of the Gray vs the Green Online indicator?

### DISCUSSION THREAD

**Discussion by vmsman** [netalertx/NetAlertX#750] [Q&A] [answered]
What is the meaning of the Gray vs the Green Online indicator?  Is this a setting?
![image](https://github.com/user-attachments/assets/185bcd1a-4b5f-4f81-87d0-be615980bddc)

**Comment by jokob-sk** [comment, marked as answer]
Hi @vmsman ,

I think if you refresh the cache the colors will get fixed. (use the 🔃 button and shit + reload the browser tab)

**Reply by vmsman** [reply]
Thanks.  I had no idea that was a stale cache.
